examples/logs/main.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_vpc" "es_vpc" {
```

Comment:
If we could use the networking module we would be avoiding inconveniences in the future imho